c++/examples/boost.rst

```diff
@@ -0,0 +1,242 @@
+
+.. code-block:: cmake
+
+    bii_find_boost([COMPONENTS component1 component2 ...] [REQUIRED])
+
+It takes the Boost components you need, exactly as ``find_package(Boost)``, configures the biicode boost installation, builds that components, and then calls ``find_package(Boost)``.
+
+biicode uses the current cmake C++ compiler as Boost toolset, multiple toolsets are supported inside the same Boost version installation. Also note that, except you delete the libraries at the biicode environment, each Boost component will be built only once for each Boost version and toolset.
+
+Build and run project
+^^^^^^^^^^^^^^^^^^^^^
+
+To configure and build your project, just run ``bii cpp:configure`` and ``bii cpp:build as usual:
+
+.. code-block:: bash
+
+	$ bii cpp:configure 
+			 block your-account/boost-lambda
+	------------------------------------------------
+	-- Setting up biicode Boost configuration...
+	-- Boost version: 1.57.0
+	...
+
+	$ bii cpp:build
+
+Boost version
+^^^^^^^^^^^^^
+
+You may noticed that the example above uses Boost 1.57.0. This is the default Boost version, shipped by the ``biicode/boost`` master track. If you want other version, say Boost 1.56.0, go to the block ``biicode.conf`` file and edit the requirements entry, explicitly asking for the ``biicode/boost`` track you want:
+
+``blocks/your-account/boost-lambda/biicode.conf``
+
+.. code-block
+
+	[requirements]
+		biicode/boost(1.56.0)
+
+.. container::infonote
+  **Tip**
+
+  You may want to support multiple tracks for your block depending on the Boost version it uses. This can be acchieved easily setting different tracks for your block, matching the Boost version requested:
+
+  .. code-block
+
+  	[requirements]
+		biicode/boost(1.56.0)
+
+	[parent]
+		your-account/boost-lambda(1.56.0): -1, 0, whatever version is available
+
+Boost.Coroutine
+---------------
+
+Boost.Coroutine implements `coroutines <http://en.wikipedia.org/wiki/Coroutine> which can be useful to implement cooperative multitasking, iterators, etc, in a more natural way.
+
+.. code-block:: bash
+
+	$ bii open examples/boost-coroutine
+
+Here's `the Boost.Coroutine example <http://www.biicode.com/manu343726/manu343726/boost-coroutine/master/0/main.cpp>:
+
+.. code-block:: cpp
+	#include <boost/coroutine/all.hpp>
+	#include <iostream>
+
+	using namespace boost::coroutines;
+
+	void cooperative(coroutine<void>::push_type &sink)
+	{
+	    std::cout << "Hello";
+	    sink();
+	    std::cout << "world";
+	}
+
+	int main()
+	{
+	    coroutine<void>::pull_type source{cooperative};
+	    std::cout << ", ";
+	    source();
+	    std::cout << "!\n";
+	}
+
+In the example above, the string "Hello, world!" is written by writting "Hello", going back to ``main()`` which writes the comma, then calling the corutine again to continue printing "world", and finally printing *!* on ``main()``.  
+
+You can visualize it as:
+
+.. code-block
```

Comment:
no se formatea el code block puedes poner :: tras You can visualize it as:: y evitar el code block o definir el code-block